posts/2022-03-17-graphql-22004-beta.adoc

```diff
@@ -0,0 +1,327 @@
+---
+layout: post
+title: "MicroProfile GraphQL 2.0, Java 18 support, and distributed security caches on Open Liberty 22.0.0.4-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: MicroProfile GraphQL 2.0, Java 18 support, and distributed security caches on Open Liberty 22.0.0.4-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.4-beta offers GraphQL 2.0, incorporating Jakarta EE 9.1 dependencies, as well as Java 18 support with many new exciting features and changes. Also included in this release is distributed security caches, allowing multiple Liberty servers to share caches via a JCache provider.
+blog_description: Open Liberty 22.0.0.4-beta offers GraphQL 2.0, incorporating Jakarta EE 9.1 dependencies, as well as Java 18 support with many new exciting features and changes. Also included in this release is distributed security caches, allowing multiple Liberty servers to share caches via a JCache provider.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MicroProfile GraphQL 2.0, Java 18 support, and distributed security caches on Open Liberty 22.0.0.4-beta
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.4-beta offers GraphQL 2.0, incorporating Jakarta EE 9.1 dependencies, as well as Java 18 support with many new exciting features and changes. Also included in this release is distributed security caches, allowing multiple Liberty servers to share caches via a JCache provider.
+
+The link:{url-about}[Open Liberty] 22.0.0.4-beta includes the following beta features (along with all GA features):
```

Comment:
it seems like "feature" is being used in two senses here, twice to mean a "Liberty feature",  and twice more to mean general new function/support- ie MP GraphQl is a feature, as are "all GA features", Java 18 is not a feature except in a general sense, and the beta feature for "Distributed Security Caches" is actually distributedSecurityCache-1.0.

Seems odd but maybe Im the only one who would notice :)

Reply:
I kept going  back and forth between this and 'features and function', but figured no one would notice and this way it was shorter.  I have no issues changing this to 
```suggestion
The link:{url-about}[Open Liberty] 22.0.0.4-beta includes the following beta features and function (along with all GA features and function):
```